src/com/intellij/idea/plugin/hybris/tools/remote/http/AbstractHybrisHacHttpClient.java

```diff
@@ -111,22 +111,50 @@ public String login(@NotNull Project project, @NotNull HybrisRemoteConnectionSet
     }
 
     @NotNull
-    public final HttpResponse post(@NotNull Project project, @NotNull String actionUrl, @NotNull List<BasicNameValuePair> params, boolean canReLoginIfNeeded) {
+    public final HttpResponse post(
+        @NotNull Project project,
+        @NotNull String actionUrl,
+        @NotNull List<BasicNameValuePair> params,
+        boolean canReLoginIfNeeded,
+        final long timeout
+    ) {
+        return post(project, actionUrl, params, canReLoginIfNeeded, 6000L);
```

Comment:
I think here is the recursion in 'post' method

Reply:
fixed